src/Model/GroundWaterEnergy/gwe-uze.f90

```diff
@@ -265,6 +270,19 @@ subroutine find_uze_package(this)
     this%idxbudtheq = this%flowbudptr%nbudterm + 1
   end subroutine find_uze_package
 
+  !> @brief Check to ensure auxiliary areas equal respective cell areas
+  !<
+  subroutine uze_chk_aux_area(this)
```

Comment:
Do we still need uze_chk_aux_area and uzarea_chk?  I'd be inclined to condense these into one routine called uze_ad_chk.

Reply:
Consolidation is always better, thanks for catching that.  addressed in 1ebcad4